modules/ROOT/pages/aggregator/aggregator-examples.adoc

```diff
@@ -23,7 +23,7 @@ So, you could use this:
 </aggregators:size-based-aggregator>
 ----
 
-If you want to execute some action every time your list of names is filled by a 10%. You do not want to do this for every new element because that would cause too many executions. You can assume that the logic to be executed is in another flow named `partialAction`. We can you the `incremental-aggregation` route for this.
+If you do not want to execute some action for every new element because that would cause too many executions, you can instead use `incremental-aggregation`. For example, you could execute an action every time your list of names increases by an increment of 10% as shown in the below example. You can assume that the logic to be executed is in another flow named `partialAction`. 
```

Comment:
Questions about "execute some action":

Did you intend "execute" or "repeat"?
Did you mean "some actions"? "an action"? " "a particular action"?

Reply:
Also, can the final sentence follow the example, so that we can introduce the example directly?
```suggestion
If you do not want to execute an action for every new element because that would cause too many executions, you can use `incremental-aggregation`. For example, you could execute an action every time your list of names increases by 10%, as shown here:

You can assume that the logic to be executed is in another flow named `partialAction`. 
```